docs/getting-started/runner.md

```diff
@@ -3,9 +3,11 @@ title: Runner
 sidebar_position: 6
 ---
 
-## A developer tool and a DApp for running contracts on the blockchain
+In this video, Nick Stanford, Developer Relations Manager for Smart Contracts, shows you how to deploy and execute contracts created in Marlowe Playground using Marlowe Runner, beta version. Marlowe Runner is a developer tool with a friendly user interface for deploying and executing smart contracts on Cardano, straight from a browser.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/B5XcH0j7Y7w?si=9e0Rf2TLo1Y2kMHp" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
nit: This needs to be inside a wrapper otherwise it will not be responsive on mobile layouts

![image](https://github.com/input-output-hk/marlowe-doc/assets/119703292/34948e92-5f36-4288-ac6b-ff59c2a0386a)

Reply:
OK, I didn't realize that. I based this layout on a similar embedded video on this page: 
https://docs.marlowe.iohk.io/docs/developer-tools/playground
Let's sort out an appropriate layout so that it works on mobile as well. I think this could wait until next week, especially since the video is available through other channels such as twitter/X.